lib/cosmos/tools/tlm_viewer/screen.rb

```diff
@@ -258,6 +258,8 @@ def process(filename)
 
       begin
         parser = ConfigParser.new("http://cosmosrb.com/docs/screens/")
+        parser.instance_variable_set(:@original_target_name, @original_target_name)
+        parser.instance_variable_set(:@target_name, @substitute)
```

Comment:
@substitute can be nil, and if so you should use @original_target_name (Though can be nil above, will always end up being set in tlm_viewer_config.rb#start_screen).